flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/benchmark/scheduling/InitSchedulingStrategyBenchmarkTest.java

```diff
@@ -44,4 +46,20 @@ public void initSchedulingStrategyBenchmarkInBatchJob() throws Exception {
         benchmark.initSchedulingStrategy();
         benchmark.teardown();
     }
+
+    @Test
+    public void initSchedulingStrategyBenchmarkInAdaptiveBatchJob() throws Exception {
```

Comment:
Yes, there is indeed no time-consuming operation at present, but other benchmarks in scheduler have introduced PipelinedRegionSchedulingStrategy and VertexWiseSchedulingStrategy, it may be better to be consistent here? At the same time, if other time-consuming operations are introduced in VertexWiseSchedulingStrategy in the future, we can also detect them in time.